### PR TITLE
May Release Notes

### PR DESCRIPTION
**Pull request by NWeber-SL**
Links don't work because corresponding new articles are in other PRs. Need you approval by EOD today!

### CODE REVIEW

File: release-notes/april-2019.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+* **Workspaces**. ShiftLeft Ocular now includes workspaces for easy management of CPGs and overlays. [Refer to the API](https://ocular.shiftleft.io/api/io/shiftleft/repl/Workspace.html) for additional information. 
+
+* **Load Multiple CPG Queries**. You can now load more than one CPG in a given workspace and then combine queries. [Refer to the API](https://ocular.shiftleft.io/api/io/shiftleft/repl/Console.html) for additional information.
+
+* **Deprecated `sp` Object**. The functionality of the deprecated `sp` object has been transferred to `cpg` object.
+
+* **Dependency List Outputs Complete Dependencies**. The Dependency List has been fixed to now output complete dependencies.
+
+* **Renamed newLocation Step**. The newLocation step has been renamed to location.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/april-2019.md]:
```suggestion
* **Renamed newLocation Step**. The `newLocation` step has been renamed to `location`.
```

File: release-notes/may-2019.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+* **Information available on Querying for Atomicity/Race Conditions for C/C++**. The documentation now includes information on using ShiftLeft Ocular to query for atomicity and race conditions. 
+
+* **New Tutorial on using ShiftLeft Ocular with JavaServer Pages (JSP)**. The ShiftLeft Ocular documentation now includes a [tutorial on using ShiftLeft Ocular to examine the software elements and flows in your JSP](../using-ocular/tutorials/jsp.md) to identify complex business logic 
+vulnerabilities that can't be scanned for automatically.
+
+## ShiftLeft Inspect and Protect 
+
+* **SOC 2 Type 1 Compliance**. ShiftLeft Inspect now analyzes, and ShiftLeft Protect secures, financial data through SOC 2 compliance. By providing specific visibility into how you are handling financial data, your customers' financial statements and financial reporting can now be protected against the exploitation of vulnerabilities. The Type 1 report assesses the design of security processes at a specific point in time.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/may-2019.md]: This paragraph seems too focused on financial data, which doesn't make sense. It should be a little more generalized. Here's how another vendor announced their SOC 2: https://auth0.com/blog/auth0-achieves-soc-2-certification/ . CC @alokshukla1978

File: release-notes/may-2019.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+* **Universal Access Token for ShiftLeft Inspect and ShiftLeft Protect** The `uploadToken` and `license` properties have been  deprecated as a non-breaking change. These two properties are replaced by a single property that can be used for both uploading a token and licensing, thereby simplifying integration in CI/CD and runtime environments. The single property is  `accessToken` in JSON and `SHIFTLEFT_ACCESS_TOKEN` in environment variables.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/may-2019.md]:
```suggestion
* **Universal Access Token for ShiftLeft Inspect and ShiftLeft Protect** The `uploadToken` and `license` properties have been  deprecated as a non-breaking change. These two properties are replaced by a single property that can be used for both uploading a token and licensing, thereby simplifying integration in CI/CD and runtime environments. The single property is  `accessToken` in `shiftleft.json` and `SHIFTLEFT_ACCESS_TOKEN` in environment variables.
```

File: release-notes/may-2019.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+* **Documentation on Using a Custom Policy with ShiftLeft Inspect**. ShiftLeft provides a database of default Policies. You can also create your own custom Policy for ShiftLeft Inspect. Documentation explains how to [create and use a custom Policy](../using-inspect-protect/inspect/custom-policies.md), illustrated by the common use case of providing application- or organization-specific rules that more accurately reflect the state of sensitive data variables.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/may-2019.md]: This seems too focused on the documentation. The bigger announcement is that we support custom policies at all.

File: release-notes/april-2019.md (hunk 1 above)
Comment (NWeber-SL) [reply release-notes/april-2019.md]: Done

File: release-notes/may-2019.md (hunk 2 above)
Comment (NWeber-SL) [reply release-notes/may-2019.md]: Done

File: release-notes/may-2019.md (hunk 3 above)
Comment (NWeber-SL) [reply release-notes/may-2019.md]: Done

File: release-notes/may-2019.md (hunk 4 above)
Comment (NWeber-SL) [reply release-notes/may-2019.md]: Done.

**Pull request merged**